For review v3 (#4)
fabiobohren/npmjsreadmes: document_capacitor.md

### Changelog

For a detailed list of changes in each version, see the [changelog](https://docs.scanbot.io/document-scanner-sdk/capacitor/changelog/?utm_source=npmjs.com&utm_medium=referral&utm_campaign=dev_sites).

## How to use the SDK

### Requirements

Check out our [documentation](https://docs.scanbot.io/document-scanner-sdk/capacitor/introduction/?utm_source=npmjs.com&utm_medium=referral&utm_campaign=dev_sites) for a full overview of our SDK's requirements.

### Install steps

To install the Scanbot Document Scanner SDK module, run the following command in your root project folder:

```
npm i capacitor-plugin-scanbot-sdk
```

### Camera permissions

Our SDK needs camera access to scan from a live camera stream.

##### Android

> Add these camera permissions to your **android/app/src/main/AndroidManifest.xml**:

Reviewer: It's one permission only.

```suggestion
Add camera permission to your **android/app/src/main/AndroidManifest.xml**:
```

> ```

Reviewer: We can use the file ext for more "stylish" snippet:

```suggestion
```xml
```

<uses-permission android:name="android.permission.CAMERA" />

<uses-feature android:name="android.hardware.camera" />
```

##### iOS

> Add this camera permission description to **ios/{projectName}/Info.plist** inside the <dict> element:

Reviewer: This is how we add the permission on iOS. I think we can use the same phrase:

```suggestion
Add camera permission to **ios/{projectName}/Info.plist** inside the <dict> element:
```

> ```

Reviewer: We can use the file ext for more "stylish" snippet:

```suggestion
```plist
```

<key>NSCameraUsageDescription</key>

<string>Camera permission is needed to scan documents</string>
```

### SDK initialization

Initialize the SDK before using any features, ideally when your app launches.

Add this import to the top of your file:

> ```

Reviewer: We can use the file ext for more "stylish" snippet:

```suggestion
```ts
```

> import ScanbotSDK from 'capacitor-plugin-scanbot-sdk'

Reviewer: ```suggestion
import ScanbotSDK from 'capacitor-plugin-scanbot-sdk';
```

```

Initialize the SDK with:
```
ScanbotSDK
.initializeSDK({ licenseKey: "" })
.then(result => console.log(result))
.catch(err => console.log(err));
```

💡 You can test the SDK without a license key for 60 seconds per app session. Need longer testing? Get your [free trial license key](https://scanbot.io/trial/?utm_source=npmjs.com&utm_medium=referral&utm_campaign=dev_sites).

### Implementing the document scanner

With our [Ready-To-Use UI (RTU UI)](https://docs.scanbot.io/document-scanner-sdk/capacitor/features/document-scanner/ready-to-use-ui/?utm_source=npmjs.com&utm_medium=referral&utm_campaign=dev_sites) components, you can start the document scanner screen with just a few lines of code:

> ```

Reviewer: We can use the file ext for more "stylish" snippet:

```suggestion
```ts
```
